dotcom-rendering/src/types/palette.ts

```diff
@@ -152,29 +152,33 @@ export type Palette = {
 
 export type ContainerOverrides = {
 	text?: {
```

Comment:
**tl;dr** I have made all these properties optional because the Guardian Labs container does not override most of these.

I think this is ok, as different containers types should be able to override different parts of the palette. I think it's less weird than providing a 'Branded' override colour for each of these, when if fact it would be the same as the default value, and if the default value would change we would want the Labs container to change too.  

Keen to hear more thoughts on this tho because it's not ideal. An option would be for example convincing the Labs team to have their entire background team, not just the left column. But this might look weird with the teal card headlines. So a more comprehensive redesign would be needed to bring this container in line with the rest.